### ISSUE TITLE
Key verses (Q/F) "spiritually significant" requirements

### ISSUE THREAD

**Issue by gryphonshafer** [gryphonshafer/Quizzing-Rule-Book#65]
In the 2018 rule book and in the current best practices document, there's a requirement that verses to be written as Q/F type questions be "spiritually significant". These rules were moved to the best practices document because they were subjective and therefore sometimes not followed. Moving them to the best practices kept the v1 rule book functionality equivalent to the 2018 rule book, but the desire is to prevent verses poorly suited for Q/F types from being used on such questions.

One possible solution is to do 3 things:

- Establish objective criteria for verse selection on a "key verses list" (KVL)
- Write a rule requiring that a KVL is generated prior to questions for a given material year
- Add additional objective rules clearing up F-subtype misuse issues

While the second and third item are easy, the first item is not easy. What follows is a draft of an idea that might work as an 80% solution for the first item. (This may need some additional edit iterations.)

- 1 verse or 2 sequential verses
- Explicitly expresses 1 or more internally identifiable tenets of Christianity
    - Either through didactic passages
    - Or through narratives that describe people following doctrine
    - Either directly or indirectly by example
- Doesn't beg an antecedent to identify each and every of its doctrinal concepts

**Comment by kclimenhaga** [comment]
I suspect that any such language will result in too narrow of a key verse list. In practice, what I do as a question writer is to assume that all verses can be included, and then only exclude them if I feel they do not meet an acceptable level of significance. This wording would require me to do the opposite, and assume that no verses can be included, then subject each verse to this test to decide whether I can include them. I think the language around this needs to be vague, because I don't think there will be any sort of criteria that would allow different question writers to come up with identical lists. 

For example:
Luke 2:5: "He went there to register with Mary, who was pledged to be married to him and was expecting a child." 
I would argue that this verse is spiritually significant, since it talks about one of the conditions necessary for the nativity (Mary and Joseph going to Bethlehem). However, according to the criteria you described, I would say it is not spiritually significant, since it does not explicitly express an identifiable tenet of Christianity, nor are Mary and Joseph following doctrine at that moment in time.

**Comment by gryphonshafer** [comment]
Conditions necessary for the nativity seems like an identifiable tenet of Christianity to me. Regardless, what I wrote I labeled as draft of an idea that might work as an 80% solution. I think it necessarily requires some iteration.

**Comment by kclimenhaga** [comment]
I agree that some question writers would call this an identifiable tenet of Christianity; for myself, it is too implicit to meet the criteria you outlined. So again, we have a situation where some question writers will include this verse, and others will not. Maybe we can come up with slightly clearer wording, but I think that no matter how clear the wording is, there will always be a level of subjectivity. I am also opposed to having an official key verse list for Quote/Finish questions, because I want quizzers to memorize all of the material, not just these particular verses. Every verse is important; it's just that some verses do not stand alone well enough to ask as quote questions.

**Comment by scottpeterson** [comment]
I don't have a strong opinion on the outcome, I just think the current world is super dumb: the current rule book lays out multiple requirements for a Q/F (albeit subjective). The ONLY reason to have such requirements are that not all verses meet them. And yet question writers will just say "I think every verse should be a Q/F" and write every verse and quizzers get tested on that 

Why would we be okay with question writers choosing their personal desire over what's written in the rule book? 

If we want different questions written, or different verses included, just change the rule book. 

If a QM personally desires more MAs in a quiz, and just puts in 6, that's fine? It's an analogous scenario, the only difference being that # of MAs is 100% objective, and the requirements for Q/F being less than 100% objective.

**Comment by gryphonshafer** [comment]
Honestly, I think I could be convinced we should remove any requirement of "spiritual significance" for Q/F. But as soon as I admit this out loud, Jeremy will remind me why that's a bad idea. (I just can't remember right now why it's a bad idea.)

Assuming it is a bad idea, then I think we need to find an objective rule. I agree that finding a good + objective rule will not be easy. It might in fact be really, really difficult. But it's not impossible.

**Comment by scottpeterson** [comment]
The rule book heavily implies to quizzers that Q/F will be subject to some "spiritual significance" by question writers, albeit subjective. So while the quizzer isn't expecting question writers to write the exact same list, the quizzer IS expecting question writers to try and apply the rule book language to the best of their ability. 

If the verses they're tested on are written by writers who did NOT try and apply the rule book language to the best of their ability, then that is demotivating for the quizzer.

**Comment by kclimenhaga** [comment]
> I think I could be convinced we should remove any requirement of "spiritual significance" for Q/F.

My initial thought was to agree, but I tried to find an example and am now less convinced. James 2:15 "Suppose a brother or a sister is without clothes and daily food." I would be disappointed to see that as a Q/FTV on its own, but it would be fine as a Q2/F2V with verse 16 "If one of you says to them, 'Go in peace; keep warm and well fed,' but does nothing about their physical needs, what good id it?" Do we just tighten the language around "complete thought" and say that 2:15 is not a complete thought?

**Comment by jswingle** [comment]
> Honestly, I think I could be convinced we should remove any requirement of "spiritual significance" for Q/F. But as soon as I admit this out loud, Jeremy will remind me why that's a bad idea. (I just can't remember right now why it's a bad idea.)

Nope, actually! I 100% think we should remove this from the rulebook. My argument being:

1. Tons of question writers completely ignore it.
2. The ones who don't ignore it can't even remotely agree with one another on how to apply it.

At _district-level quizzing_ I think it is excellent and on-mission to have a subset of spiritually significant key verses. It serves an important pedagogical purpose (focusing on the most significant content for quizzers who don't memorize full material) and reduces the difficulty of quizzing by only requiring a certain subset of verses to ever be quoted word-perfect.

At _higher levels of quizzing, especially Internationals_, every quizzer should just know every verse word perfect anyways. I don't think it serves our mission to require the quizzers to guess as to the question writer's opinion of what is "spiritually significant" or not. The range of what constitutes "spiritual significance" changes DRAMATICALLY from one year of Internationals to the next. I've seen some truly absurd stuff get asked as Finish/Quote, and it drives me nuts that I can't throw them out because there's no objective criteria on which to do so. So let's just get rid of the rule, and encourage districts to set their own key verse lists if they think it is the best approach to encourage their quizzers.

I appreciate the attempt to create a more objective criteria, but I'll be brutally honest as to what I think would happen: question writers would all disagree on how to apply the super well-thought-out criteria. Attempting to define what makes one verse more spiritually significant than another _in the Bible_ is always going to be like trying to nail jello to a tree.

**Comment by scottpeterson** [comment]
I could not agree more.

**Comment by gryphonshafer** [comment]
I'm in favor of the conclusion of Jeremy's most recent comment here, but I still maintain there is an objective solution discoverable that wouldn't be tree jello.

Also, if we remove this rule completely, we should be prepared for the wrath of Kristine. And to be fair, she has a point. Verses that are sentence fragments getting asked as Q/Fs is weird, even though I agree with Jeremy that by the IBQ level, quizzers should have no problems with such questions.

**Comment by kclimenhaga** [comment]
This goes back to my earlier comment about "complete thoughts". I agree that Romans 1:2-3 is fine as a Q2/F2V, but there are some verses that I would not want to see as Q/FTV even at IBQ. The example that comes to mind is Romans 1:31 in the ESV: "foolish, faithless, heartless, ruthless". It doesn't have to be a complete sentence to be a complete thought; if it did, we couldn't quote Romans 3:23. But I think it should be a complete thought to be used as a quote. Maybe that language is still too vague, but I would like to see something along those lines at least as a best practice.

**Comment by jswingle** [comment]
I agree, Rom 1:31 definitely shouldn't be a Q/FTV. This is a tough scenario, though, because the rules already prohibit many of these questions that still sneak in anyway because the criteria is too vague and gray to make an objective decision. That's why I feel that it would be better to get rid of the rule altogether - at least quizzers wouldn't be _surprised_ when weird verses show up.

**Comment by jswingle** [comment]
Issue #113 concerns rule variations and how they might be notated within the rulebook. I think this could also apply to a Best Practices document.

For example, we could indicate a recommendation that at the district level, Finish/Quote questions be used pegagogically to highlight those verses which district leadership considers most spiritually significant, and the option of using official KVLs or Club Lists to this effect.

However, I think the language of "spiritual significance" should entirely be removed from what is expected out of IBQ best practices. I have two lines of reasoning, both of which are practical:

1. Nobody agrees on what constitutes spiritual significance, and there are ALWAYS people unhappy with the choices of the question set every year at IBQ. (I'm often one of those people.)
2. Quizzers already need to know the entire material (to some degree) to be actually competitive at IBQ, so limiting the question set like this doesn't seem to serve any missional purpose. 
 
Why keep a "best practice" that is to no advantage and causes frustration and disagreement?

**Comment by jswingle** [comment]
However, even if we remove spiritual significance from the IBQ best practices, we still need to keep the guidelines for what makes a good Q2V/F2V/FTN. Much like spiritual significance, these rules are subjective and cannot really be challenged, yet they are still in the rulebook instead of the best practices document. I would suggest we remove these bullet points from the rulebook and instead write from scratch a set of Best Practices guidelines for selecting Q/F subtypes. If we have agreement that these should be moved out of the main rulebook, I'd be happy to draft such a Best Practices section.

1.2.2.4.2. Quote 2 Verses (Q2V)
Remove the sentence "Q2V is used when a verse is not strong enough on its own and needs another verse to explain it or can be used when 2 verses combine to form an even stronger thought"

1.2.2.4.6 Finish 2 Verses (F2V)
Remove the sentence "Used when a verse is not strong enough on its own and needs another verse to explain it or can be used when 2 verses combined, form an even stronger thought"

1.2.2.4.7 Finish This and the Next (FTN)
Remove the sentence "Used when part of a verse is not strong enough on its own and needs another verse (or part of a verse) to explain it or when 2 verses combined form an even stronger thought"
